stable/yourls/templates/deployment.yaml

```
@@ -0,0 +1,180 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "yourls.fullname" . }}
+  labels:
+    app: "{{ template "yourls.fullname" . }}"
+    chart: "{{ template "yourls.chart" . }}"
+    release: {{ .Release.Name | quote }}
+    heritage: {{ .Release.Service | quote }}
+spec:
+  selector:
+    matchLabels:
+      app: "{{ template "yourls.fullname" . }}"
+      release: {{ .Release.Name | quote }}
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: "{{ template "yourls.fullname" . }}"
+        chart: "{{ template "yourls.chart" . }}"
+        release: {{ .Release.Name | quote }}
+      {{- if or .Values.podAnnotations .Values.metrics.enabled }}
+      annotations:
+      {{- if .Values.podAnnotations }}
+{{ toYaml .Values.podAnnotations | indent 8 }}
```

If you don't target versions older than helm 2.7 (released Oct 24, 2017), you might use the following syntax for cleaner indentation in your template (it shouldn't change the output)
```suggestion
        {{- toYaml .Values.podAnnotations | nindent 8 }}
```

---

> If you don't target versions older than helm 2.7 (released Oct 24, 2017)

That's a good point. Do you think this is safe enough? (@darend?)

---

Oh and is it normal to add the dash?

---

> That's a good point. Do you think this is safe enough? (@darend?)

👍 

> Oh and is it normal to add the dash?

In this context, yes

---

@darend Thanks! Changed.